gui/system/forms.py

```diff
@@ -998,19 +998,22 @@ def __init__(self, *args, **kwargs):
 
         self.fields['stg_language'].choices = settings.LANGUAGES
         self.fields['stg_language'].label = _("Language (Require UI reload)")
-        self.fields['stg_guiaddress'] = forms.ChoiceField(
-            label=self.fields['stg_guiaddress'].label
+        self.fields['stg_guiaddress'] = forms.MultipleChoiceField(
+            label=self.fields['stg_guiaddress'].label,
+            required=False
         )
-        self.fields['stg_guiaddress'].choices = [
-            ['0.0.0.0', '0.0.0.0']
-        ] + list(choices.IPChoices(ipv6=False))
+        ipv4_choices = list(choices.IPChoices(ipv6=False))
+        ipv6_choices = list(choices.IPChoices(ipv4=False))
```

Comment:
Why are you removing wildcard from choices?

Reply:
So the wildcards are being added below in the choices. This needed to be done to ensure that UI doesn't display duplicates in ip choices